uk.co.compucorp.civicrm.hremails/email-templates/src/assets/scss/components/_request.scss

```diff
@@ -15,12 +15,17 @@
 
 
 .request-comment {
-  padding: $callout-padding;
+
+  &-body {
+    padding: $callout-padding;
```

Comment:
i preferred to put the exception below to make it more readable
(btw it would be `padding: $callout-padding $callout-padding 0 $callout-padding`)